themes/ee/asset/javascript/src/cp/grid.js

```diff
@@ -259,6 +260,13 @@ Grid.Publish.prototype = Grid.MiniField.prototype = {
 		if (this.isFileGrid) {
 			this.root.toggleClass('hidden', rowCount == 0)
 		}
+
+		if(rowCount == 0) {
+			setTimeout(function(){
```

Comment:
Is there a specific need for `setTimeout` here? (thinking because the code above does not use it)

Reply:
we need this `setTimeout`, since the element is being replaced and we need to catch and fix the new element

Reply:
In such case, is `50` milliseconds sufficient? Or maybe we need to check for actual elements existance (in a loop with timeout)

Reply:
I agree with Yuri. 50 ms is 1/20th of a second. That seems to be such a small amount of time that I would think it would still fail that check on if the page with a lot of other elements to load or if the user has a slower computer. I think this needs to be in a setInterval loop with a clearInterval when it's found.

Reply:
ok, I changed setTimeout to setInterval with clearInterval